api/lib/routes/metrics.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+module.exports = function(settings) {
+  var express = require("express");
+  var router = express.Router();
+  var logger = require("../log/logger");
+  var HttpStatus = require("http-status-codes");
+  var metricsCollectorUtil = require("../utils/metricsCollectorUtils")(settings.metricsCollector);
+  var metricHelper = require("./metricHelper");
+  var routeHelper = require("./routeHelper");
+
+  router.get("/:app_id/metrics", function(req, resp) {
```

Comment:
@boyang9527 in our api document, metricType is a query parameter. Here need your confirm we set it to path variable.